src/routes/v1/article.route.js

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import ArticleController from '../../controllers/articles.controller';
+import validationMiddleware from '../../middleware/validation.middleware';
+import paginationValidations from '../../middleware/pagination.validation';
+
+const validateRequest = validationMiddleware();
+
+const router = Router();
+
+router.get(
+  '/articles',
```

Comment:
Good job! here. I think you can make this route this same as the user routes. Let the get request be '\' while the 'articles' segement is added to the route index.